daffodil-io/src/main/scala/org/apache/daffodil/io/StringDataInputStreamForUnparse.scala

```diff
@@ -66,6 +64,9 @@ final class StringDataInputStreamForUnparse
   override def getSomeString(nChars: Long,finfo: FormatInfo): Maybe[String] = dis.getSomeString(nChars, finfo)
   override def getString(nChars: Long,finfo: FormatInfo): Maybe[String] = dis.getString(nChars, finfo)
 
+  // $COVERAGE-OFF$ Nothing should be calling these.
```

Comment:
Highlight: This coverage-on/off stuff works. I got the coverage on this change set to pass codecov's criteria by putting these things around code that shouldn't have coverage.